functions/api/dashboardFunctions.ts

```diff
@@ -2,10 +2,25 @@ import { Request, Response } from 'express';
 import { error } from 'firebase-functions/logger';
 import { recalculatePlayerAverages } from './scheduled/recalculatePlayerAverages';
 import generateLeagueAverage from './scheduled/generateLeagueAverage';
+import { returnAuthToken } from '../src/auth';
+import { Audit } from '../types/audits';
+import { addAudit } from '../utils/addAudit';
+import admin from 'firebase-admin';
```

Comment:
absolute imports should always be above relative ones. Absolute imports are when you're not going directly from this file `../etc.` and you import directly like `file`. Basically whenever you don't import starting with `../` or `./`